.github/workflows/continuous-integration.yml

```diff
@@ -18,59 +18,55 @@ env:
 jobs:
   test:
     name: Test
-    timeout-minutes: 15
+    timeout-minutes: 20
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [14, 16, 18, 19]
+        node-version: [16, 18, 19]
         include:
+          - os: windows-latest
+            node-version: 18 # LTS
           - os: macos-latest
             node-version: 18 # LTS
       fail-fast: false
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3
```

Comment:
AFAIK, the version tag comment is still required for renovate:

https://github.com/loopbackio/loopback-next/pull/8868

Side note: The syntax seems to have changed to be even more terse:

https://github.com/renovatebot/renovate/pull/18738

Example:
https://github.com/loopbackio/loopback-next/commit/df23dfc99699cb8aa2f61926b128b5f3ac3e41ea

Reply:
I had no idea, I'll change it. Thanks for the information